content/events/workshop-amsterdam-2020-03-30.md

```diff
@@ -0,0 +1,55 @@
+---
+title: KubeCon EU  Pre-day Workshop
+url: /events/workshop-amsterdam-2020-03-30
+type: page
+layout: event-single
+calendly_url: "https://calendly.com/pulumi/kubecon?month=2020-03&hide_event_type_details=1&primary_color=512668"
+---
+## Details
+
+<div>
+	<i class="fas fa-globe-americas inline-block text-xl"></i>
+	<span class="my-0 text-xl">
+	    <a href="https://maps.google.com/?q=Europaboulevard 2b, 1078 RV Amsterdam, Netherlands" class="text-blue-500">Europaboulevard 2b, 1078 RV Amsterdam, Netherlands</a>
+	</span>
+</div>
+
+<div>
+	<i class="fas fa-calendar inline-block text-xl"></i>
+	<span class="my-0 text-xl">March 30, 2020</span>
+</div>
+
+<div>
+	<i class="fas fa-clock inline-block text-xl"></i>
+	<span class="my-0 text-xl">11:30AM - 4:00pm (CET)</span>
+</div>
+
+<div>
+	<i class="fas fa-ticket-alt inline-block text-xl"></i>
+	<span class="my-0 text-xl">€99</span>
+</div>
+
+Note: The registration fee includes lunch, the workshop and happy hour afterwards.
+
+The hardest part of Kubernetes is setting up the infrastructure: clusters, DNS, firewalls, load balancers, IAM, storage, logging, and performance monitoring, often spanning private, public, and hybrid cloud architectures.
+
+In this workshop, the Pulumi team will show you how to tackle these challenges using Infrastructure as Code (IaC) through a series of hands-on labs. The techniques work for any cloud - AWS, Azure, and GCP. You'll be able to leverage your favorite languages including Python, Go, JavaScript, TypeScript, and C# instead of YAML or domain-specific languages.
+
+After completing this workshop, you'll be up and running with IaC fundamentals, modern application architectures across many clouds, and Kubernetes best-practices that are ready for production environments. You'll also be ready to empower your development teams to be more productive - continuously deploying both their applications and infrastructure.
+
+**Agenda**
+
+11:30am - Lunch will be served in the meeting room.
+
+12:00-3:00pm - Workshop with hands-on labs.
+
+3:00-4:00pm - Happy hour with drinks/snacks.
+
+**Instructors**
+
+Joe Duffy - CEO, Pulumi <a href="https://twitter.com/@funcofjoe" target="_blank">@funcofjoe</a>
```

Comment:
Don't worry about it for this PR, but it might be nice to link the persons name to the "authors" section on the blog, e.g. https://www.pulumi.com/blog/author/joe-duffy/. Maybe useful?

Reply:
Yeah, at some point in the future we will want to add pictures of the instructors and maybe more contact info.